docs/topics/whatsnew18.md

```diff
@@ -39,48 +39,48 @@ The new language version also includes:
 
 ### Ability to not generate TYPE_USE and TYPE_PARAMETER annotation targets
 
-If a Kotlin annotation has `TYPE` among its Kotlin targets, the annotation maps to `java.lang.annotation.ElementType.TYPE_USE` 
-in its list of Java annotation targets. This is just like how the `TYPE_PARAMETER` Kotlin target maps to 
-the `java.lang.annotation.ElementType.TYPE_PARAMETER` Java target. This is an issue for Android clients with API levels 
+If a Kotlin annotation has `TYPE` among its Kotlin targets, the annotation maps to `java.lang.annotation.ElementType.TYPE_USE`
```

Comment:
Small tip for the future: I'd really recommend committing such "automatic clean-up" changes in a separate commit, as that really simplified review and, later on, reading commit history. in `kotlin.git` or `intellij.git` we always do that, and actually almost require people to commit "code reformatting" separately from the semantical changes

Reply:
Sorry, I also hate to see this, but our documentation plugin did it automatically.